lib/lotus/commands/db/console.rb

```diff
@@ -0,0 +1,54 @@
+require 'lotus/utils/class'
+
+module Lotus
+  module Commands
+    class DB
+      class Console
+        attr_reader :name, :env_options, :environment
+
+        def initialize(name, environment)
+          @name        = name
+          @environment = environment
+          @env_options = environment.to_options
+          load_config
+        end
+
+        def start
+          exec connection_string
+        end
+
+        private
+
+        def config
```

Comment:
Cool I like it. I have theses methods too. We should extract it to reuse. Since we're going to create an especific class to migrations